tools/jenkins/testSuites/dev/regression-test-part-5-Jenkinsfile

```diff
@@ -70,5 +79,13 @@ pipeline {
         }
       }
     }
+    stage('delete school principal') {
+      steps {
+        dir('frontend') {
+          sh 'npm install'
+          sh 'npm run deleteSchoolPrincipal'
```

Comment:
Similar to above. We probably need to delete the school that was created.

Reply:
Oh dang that's a really good point. We'll have to think of a solution around that eventually.